### DISCUSSION TITLE
[Feature Suggestion] Allow user to toggle dark/light mode via settings

### DISCUSSION THREAD

**Discussion by florianfelsing** [AllYourBot/hostedgpt#168] [General]
Currently the system settings determine whether users get to see our application in light mode or dark mode. 

I think it would be nice to allow them to set their preference somewhere in the settings. Personally, I like running my system in light mode, while running most of my apps in dark mode.

**Comment by keithschacht** [comment]
Good call. I’ll add this to the list. I’ll put it on the v1.0 list but if this is a feature that would matter a lot to you, feel free to tackle it now! It should be a pretty small one. Otherwise, I’ll put it on the list for next rev since we still have meaningful feature gap with ChatGPT.

I keep asking myself: “what features will probably most lead to others using this daily as their go-to over chatgpt?” But anything that matters for *you* or me to use it daily, that gets top priority :) since the more we use it, the faster we’ll make the overall app better.

(BTW,  I just realized I had notifications disabled for new discussions! I’m catching up on things now and I fixed my notification settings.)

**Reply by florianfelsing** [reply]
Nah, it's fine - it is not _that_ important to me in terms of using HostedGPT over ChatGPT. 

To me it's more one of those features that on its own don't make much of a difference, but can be really powerful once they accumulate.
